evm/bridge-token-factory/contracts/BridgeTypes.sol

```diff
@@ -27,16 +27,16 @@ library BridgeTypes {
         address indexed sender,
         address indexed tokenAddress,
         uint128 indexed nonce,
-        string token,
         uint128 amount,
         uint128 fee,
         uint128 nativeFee,
-        string recipient
+        string recipient,
+        string message
     );
 
     event FinTransfer(
         uint128 indexed nonce,
-        string token,
+        address tokenAddress,
```

Comment:
The relayer should be fixed as well: https://github.com/Near-One/omni-bridge/blob/main/omni-relayer/src/startup/eth.rs#L41C1-L41C22

Reply:
The relayer is out of the scope.
CC @frolvanya